.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/220867961-2c220f8c-3d71-4d0e-b5b9-8e72fd246f2e.png)

- [ ] It would be nice if you make your pull request title such as **RoR capstone project: Budget App Review** instead of **Feature branch** because your Pull Request Title is not descriptive enough :fearful: Kindly make it descriptive to make your pull request professional and to follow the requirements :+1: